apps/extension/src/ui/domains/Staking/Bond/BondForm.tsx

```diff
@@ -275,16 +286,56 @@ export const AmountEdit = () => {
             <div className="text-brand-orange line-clamp-2 text-center text-xs">
               {inputErrorMessage}
             </div>
+            {stakeWarningMessage && (
+              <div>
+                <Tooltip>
+                  <TooltipTrigger asChild>
+                    <div className="text-alert-warn flex items-center justify-center gap-1 whitespace-nowrap text-xs">
+                      <InfoIcon className="inline-block" /> {t(stakeWarningMessage)}
+                    </div>
+                  </TooltipTrigger>
+                  <TooltipContent>
+                    <div className="max-w-[26rem]">
+                      <div>
+                        {t(
+                          "The Tao network requires a wait period of 360 blocks since your last stake / unstake action.",
+                        )}
+                      </div>
+                      <div>{t("Please try again later.")}</div>
+                    </div>
+                  </TooltipContent>
+                </Tooltip>
+              </div>
+            )}
           </div>
         </>
       )}
     </div>
   )
 }
 
-const NomPoolsApr = () => {
-  const { token } = useNomPoolBondWizard()
-  const { data: apr, isLoading } = useStakingAPR(token?.chain?.id)
+const StakeApr = () => {
+  const { token, poolId } = useBondWizard()
+  let data,
+    isLoading = false,
+    apr = 0
+
+  const hookMap = {
+    nominationPool: useStakingAPR,
+    bittensor: useGetBittensorValidator,
+  }
+
+  switch (token?.chain?.id) {
+    case "bittensor":
+      ;({ data, isLoading } = hookMap["bittensor"](poolId as unknown as string))
```

Comment:
Not such a big fan of this syntax for this, but either way would prefer to include the `isError`state here so we can show an appropriate message